crates/ruff_python_parser/src/string.rs

```diff
@@ -93,6 +86,33 @@ impl<'a> StringParser<'a> {
         TextRange::new(start_location, self.location)
     }
 
+    /// Returns the next byte in the string, if there is one.
+    ///
+    /// # Panics
+    ///  - When the next byte is a part of a multi-byte character.
```

Comment:
nit: for consistent styling
```suggestion
    /// # Panics
    ///
    /// When the next byte is a part of a multi-byte character.
```